CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## v0.51.0
+- **Allow periods in arguments to be ignored when parsing cacheKeys**: If your query arguments include periods they will no longer cause broken cache keys. This means the cached data for those queries can be correctly found and returned. [#2057](https://github.com/apollographql/apollo-ios/pull/2057) - _Thanks to [Hesham Salman](https://github.com/Iron-Ham) for the contribution!_
```

Comment:
Note that if you already have an on-device cache, you may have to delete old (invalid) cache keys to fully benefit from this change.

Reply:
Good point, I'll add that to the changelog - thanks!

Reply:
Getting back to this now and thinking about it a bit more the risk of not deleting those old-style cache keys is duplicated data. Any new request would be a cache miss and then a refetch of that data, depending on the caching policy. Do you agree with this @Iron-Ham?

Reply:
Yup, that sounds exactly right @calvincestari.
Thinking about it some more -- on our end, we needed the cache-clear because of what we were doing _outside_ of Apollo.